src/main/scala/com/typesafe/conductr/sbt/console/Column.scala

```diff
@@ -65,19 +69,33 @@ object Column {
     /**
      * Add '... ' if string is over the allowed width.
      */
-    private def ellipsize(s: String): String = {
+    protected def ellipsize(s: String): String = {
       val ellipsis = "... "
-      if (s.visibleLength >= width)
+      if (s.visibleLength > width)
         s.take(width - ellipsis.length) + ellipsis
       else
         s
     }
   }
 
+  /**
+   * The wrapping column wraps the text instead of cutting it based on the column length.
+   * At the end it uses a new line as a separator so that the next row isn't displayed on the same line.
+   * If the text is longer than the width of the column, two additional line as separation are added. The first
+   * new line is necessary so that the next row is displayed on a new line. The second new line has been added
+   * for readability reasons. On the terminal screen only one line as a separation is visible.
+   */
+  trait WrappingColumn extends RegularColumn {
+    override protected def ellipsize(s: String): String =
+      if (s.visibleLength > width)
+        s + "\n\n"
+      else
+        s
+  }
```

Comment:
Simply that when we wrap, instead of starting the next line in the first column position, we start it aligned with the start of the column.

i.e. not

```
col1 col2
     this is some
text that flows
```

...but

```
col1 col2
     this is some
     text that flows
```